microsoft-edge/devtools-guide-chromium/javascript/reference.md

```diff
@@ -366,6 +366,20 @@ If you find yourself running the same debug code in the **Console** tool over an
 See [Run snippets of JavaScript on any webpage](./snippets.md).
 
 
+<!-- ====================================================================== -->
+## Use `//# sourceURL` to name evaluated files in the Sources tool
+
+The `//# sourceURL` pragma, such as `//# sourceURL=file-name.js`, is a special comment that makes it easier to work with evaluated JavaScript code and JavaScript code found in `<script>` tags in an HTML document.
+
+When JavaScript code is run by the browser, DevTools can only name it if it was loaded from a file. When running JavaScript directly from a `<script>` tag or by using the `eval()` function, the **Sources** tool doesn't have a file name to display these files in the **Navigator** pane.
```

Comment:
```suggestion
When JavaScript code is run by the browser, DevTools can only name a block of code if the code was loaded from a file. When running JavaScript directly from a `<script>` tag or by using the `eval()` function, the **Sources** tool doesn't have a file name to display these files in the **Navigator** pane.
```
@captainbrosset - pls try to reword to be more specific and direct than "it".  To what does "it" refer?